data/remnant missions.txt

```diff
@@ -139,6 +139,99 @@ mission "Remnant Blood Test Patch"
 
 
 
+mission "lost in ember waste"
+	landing
+	invisible
+	to offer
+		has "remnant blood test pure"
+		not "license: Remnant"
+	on offer
+		attribute "quantum keystone" 0
+		require "Jump Drive" 0
+		event "lost in ember waste" 14
+		fail
+
+event "lost in ember waste"
+
+
+
+mission "Remnant: Lost in Ember Waste"
+	landing
+	invisible
+	to offer
+		has "event: lost in ember waste"
+		outfit "Quantum Key Stone" 1
+		log `Became stuck in the Ember Waste, but was given a quantum key stone by an old woman who claims to have once been stuck in Republic space under similar circumstances.`
+		conversation
+			`You wander the <origin> spaceport feeling very lost in this strange region of space. Having flown around for a few weeks with no idea of how to get out, you wonder if you'll be stuck here forever.`
+			`	You sit down on a bench and think about how you're going to get out of this situation. While you think, an old woman approaches you and signs something.`
+			choice
+				`	"Sorry, I don't understand."`
+			`	The old woman chuckles to herself. "You seem lost." She says. You nod. To your surprise, she doesn't sing everything she says like the other Remnant.
+			`	"I'm guessing you don't know about the wormholes."`
+			choice
+				`	"The what?"`
+				`	"I do, I just can't get through them."`
+					goto know
+			
+			`	"There are red wormholes scattered around this region of space. You can only access them if you have a key stone on your ship, though. Don't ask me how it works, but that's how it is."`
+				goto keystone
+			
+			label know
+			`	"Get scammed out of your key stone?" she asks with another chuckle. "Or maybe your jump drive, and now you're stuck here? No problem."`
+			
+			label keystone
+			`	The woman reaches into a bag that she has with her and hands you a quantum key stone. "Just put this on your ship and you can be on your way out of here."`
+			choice
+				`	"You're just giving this to me?"`
+					goto thanks
+				`	"What's the catch?"`
+			`	She chuckles again. "No catch."`
+			
+			label thanks
+			`	The woman sits down on the bench next to you. "I was once lost in Republic space for many years after losing my key stone. I know what it can be like to be away from your people for that long, so I'd rather not see you share the same fate."`
+			choice
+				`	"Thanks for the help. I'll go fit this on my ship."`
+				`	"How'd you get to Republic space."`
+					goto republic
+			
+			`	You say goodbye to the woman and begin walking to your ship. "Safe travels, Captain," she sings with a big grin on her face as you walk away.`
+				decline
+			
+			label republic
+			`	"I was an adventurous spirit in my youth," she says. "One day I found my way out of the Ember Waste and back to what I learned was the rest of humanity. For years my people had hidden here believing the Alphas had killed everyone else, but I now knew that we had little to fear.
```

Comment:
"...now knew..." is super awkward (mixing tenses) - ...found out...then knew...discovered...now know... are much better alternates